x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -932,7 +932,12 @@ public enum Cap {
         /**
          * Support for sorting when aggregate_metric_doubles are present
          */
-        AGGREGATE_METRIC_DOUBLE_SORTING(AGGREGATE_METRIC_DOUBLE_FEATURE_FLAG);
+        AGGREGATE_METRIC_DOUBLE_SORTING(AGGREGATE_METRIC_DOUBLE_FEATURE_FLAG),
+
+        /**
+         * Support and enable partial results by default
+         */
+        ESQL_PARTIAL_RESULTS;
```

Comment:
I see that all other capabilities do not have `ESQL` prefix, should we keep this pattern here?

Reply:
Also, we already have `SUPPORT_PARTIAL_RESULTS` capability - if those are different, we probably need to document how those partial results are different from this one.

Reply:
I've removed the new cap in https://github.com/elastic/elasticsearch/pull/125060/commits/cbb208ae63bc4e48f9247a139120b389170c4940.